roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
+
+  - name: Install docker-py as a workaround for Ansible issue
```

Comment:
I am curious what is the ansible issue: is it to allow `image` and other tags to be specified in ansible?